js/background.js

```diff
@@ -236,20 +236,19 @@ function RestoreCollection(collectionIndex, removeCollection)
 			{
 				url: i,
 				active: false
-			}/* , function (tab)
+			} , function (createdTab)
 		{
-			if (localStorage.getItem("loadOnRestore") == "false" ? true : false)
-			{
-				setTimeout(function()
-				{
-					chrome.tabs.get(tab.id, function(tab1)
-					{
-						console.log(tab.url);
-						chrome.tabs.discard(tab1.id);
-					});
-				}, 1000);
+			if (localStorage.getItem("loadOnRestore") == "false" ? true : false) {
```

Comment:
I saw you left a commented part for the implementation of this feature, so I used that as a base. One thing that bugged me however is this : I wouldn't have done this myself, but considering you left it as a comment I wanted to have your confirmation beforehand.

```suggestion
			if (localStorage.getItem("loadOnRestore") == "false") {
```